server/webapp/WEB-INF/rails.new/app/views/navigation_elements/_dropdown_admin.html.erb

```diff
@@ -0,0 +1,28 @@
+<li role="presentation">
+  <%= link_to l.string("PIPELINES_POPUP_TITLE"), main_app.pipeline_groups_path -%>
+</li>
+<%= render :partial => 'navigation_elements/dropdown_template_admin.html' -%>
+<li role="presentation">
+  <%= link_to l.string("PIPELINES_SNIPPET"), main_app.config_view_path -%>
```

Comment:
Please drop any `l.string` in the view. There's no need for any partial internationalization.